src/crimson/os/seastore/omap_manager/btree/omap_btree_node.h

```diff
@@ -0,0 +1,100 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+#pragma once
+
+#include <string>
+#include <vector>
+
+//#include <boost/iterator/counting_iterator.hpp>
+
+#include "crimson/common/log.h"
+#include "crimson/os/seastore/seastore_types.h"
+#include "crimson/os/seastore/transaction_manager.h"
+#include "crimson/os/seastore/omap_manager.h"
+#include "crimson/os/seastore/omap_manager/btree/omap_types.h"
+
+namespace crimson::os::seastore::omap_manager{
+
+struct omap_context_t {
+  omap_root_t &omap_root;
+  TransactionManager &tm;
+  Transaction &t;
+};
+
+enum class mutation_status_t : uint8_t {
+  SUCCESS = 0,
+  SPLITTED = 1,
```

Comment:
should be SPLIT;  I think Sam asked for this in the other PR that contained this code

Reply:
sorry, earlier version